pom.xml

```diff
@@ -14,20 +14,18 @@
   <name>Jenkins Distributed Fork plugin</name>
   <description>Turns a Jenkins cluster into a general purpose batch job execution environment through an SSH-like CLI.
                This plugin adds a new command 'distfork' to Jenkins CLI, which can be used to execute arbitrary command
-               on a agent of your choice. The distfork command is modeled after ssh, but it's Jenkins aware — for example,
+               on a agent of your choice. The distfork command is modelled after ssh, but it's Jenkins aware — for example,
                instead of hardcoding a machine name, you can specify a label to let Jenkins choose a agent.
                This opens up a Jenkins cluster for doing all kinds of things without requiring a job/build notion.
   </description>
-  <url>https://wiki.jenkins.io/display/JENKINS/DistFork+Plugin</url>
+  <url>https://plugins.jenkins.io/distfork/</url>
 
   <properties>
-    <jenkins.version>2.60.3</jenkins.version>
-    <java.level>8</java.level>
-    <jenkins-test-harness.version>2.34</jenkins-test-harness.version>
+    <jenkins.version>2.361.4</jenkins.version>
```

Comment:
`2.361` is too low for `matrix-auth` and some others, so bumped to the `.4` release.